docs/specs/validation-engine.md

```
@@ -370,9 +370,10 @@ runtime surface. ADR-0021 is architecture non-exclusion and does not create an
 implementation gap that blocks the strongly typed Game Development MVP.
 
 The current runtime/API surface retains snapshot validation inside the #93
-resident state owner. ADR-0022 accepts resident runtime ownership while #94–#95
-retain later selective-query/invalidation and retained-incremental
-implementation.
+resident state owner. Under ADR-0022, Issue #94 selectively projects
+stable-subject diagnostics and exact-publication invalidation derived from
+fresh dependency oracles while retaining full validation/calculation oracles;
+#95 retains incremental implementation.
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)